packages/nx/src/tasks-runner/task-orchestrator.ts

```diff
@@ -388,7 +388,15 @@ export class TaskOrchestrator {
         task,
         this.projectGraph
       );
-      if (
+
+      if (targetConfiguration.executor === 'nx:noop') {
```

Comment:
Was this important?

Reply:
bit of a leftover from when I was working there but it makes more sense to me to handle special cases first and then the general cases... I can remove it though.

Reply:
I reverted the change anyways